tratto/src/main/java/star/tratto/token/restrictions/RestrictionsUtils.java

```diff
@@ -88,15 +88,15 @@ public static List<MethodUsage> getApplicableMethodsOfPrecedingExpr(EObject last
             matchingClasses.forEach(c -> matchingMethods.addAll(
                     getMethodsOfType(fullyQualifiedClassName(c.getValue1(), c.getValue0()))
                             .stream()
-                            .filter(m -> m.getName().equals(methodName) && isStaticNonVoidNonPrivateMethod(m))
+                            .filter(m -> m.getName().equals(methodName) && isStaticNonPrivateNonVoidMethod(m))
```

Comment:
Compared to the method call just below, this one's name puts "static" before "private" and the other puts "private" before static.  Please be consistent.

Reply:
@AML14